<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chooseFurnaceDialogClass</class>
 <widget class="QDialog" name="chooseFurnaceDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>chooseFurnaceDialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>452</width>
     <height>226</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="text">
         <string>Odśwież</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sortLabel">
        <property name="text">
         <string>Sortowanie:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sortBox">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Producent A - Z</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Producent Z - A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Model A - Z</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Model Z - A</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTableWidget" name="furnaceTable">
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>0</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string>Producent</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Model</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Czy wspierany</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>chooseFurnaceDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
